socrates_gui/static/js/assets.js

```diff
@@ -39,7 +39,7 @@ export class AssetView extends React.Component {
             this.setState({key: key});
     }
     componentDidMount() {
-        if (typeof this.asset !== "undefined")
+        if (typeof this.asset !== "undefined" && this.asset.hasOwnProperty('oob'))
```

Comment:
This will break checking the power status of VMs, as they don't have an oob property. This should probably be done in socrates_api instead, in the ipmi_power_state task.

Reply:
That's what I get for trying to quickfix an issue when I was still upset about it happening :)
Not a single thought about VMs since it was an ipmi task, I'll fix it in the task itself.